src/games/kalambury/board/GameBoard.js

```diff
@@ -24,6 +37,13 @@ const GameBoard = ({
   rawClient,
 }) => {
   const [lines, setLines] = useState([]);
+  const [lastSuccess, setLastSuccess] = useState(false);
+  const lastUserGuess = getUserActions(actions, playerID, "guess")[0] || { id: null, success: false }
+
+  useEffect(() => {
+    setLastSuccess(lastUserGuess.success);
+    return () => setLastSuccess(false); // Do cleanup of state 
+  },[lastUserGuess.id]);
```

Comment:
only one comment, add `lastUserGuess.success` to `useEffect` dependencies, as it causes warning because you are using it in the function and it's not in the dependencies list